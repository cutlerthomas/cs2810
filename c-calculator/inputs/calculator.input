What is 6 added to 7?
What is -34 added to 102314?
What is 1 added to 3??
What is 5 subtracted from 7?
What is -12 subtracted from -5?
What is 4 subtracted from 9?.
What is 12 times 12?
What is 3 times 97?
What is -4 times -8?
What is 4 times -3?
What is 0 times 2?
What is 1 times 5??
What is 100 divided by 10?
What is 256 divided by 8?
What is 256 divided by 9?
What is 3 divided by 4?
What is 4 divided by 4?
What is 15 divided by 0?
What is 0 divided by 15?
What is 4 divided by 9??
What is 3 divided by 0??
What is the remainder of 20 divided by 7?
What is the remainder of 7 divided by 20?
What is the remainder of -56 divided by -17?
What is the remainder of 56 divided by -17?
What is the remainder of 56 divided by 0?
What is the remainder of 0 divided by 3?
What is the remainder of 4 divided by 4?[
What is 16 negated?
What is -99 negated?
What is 0 negated?
What is 4 negated?:
What is 16 squared?
What is -99 squared?
What is 0 squared?
What is 4 squared?:
Which is larger: 100 or 1000?
Which is larger: 999 or 9?
Which is larger: 16384 or 16384?
Which is larger: -384 or -128?
Which is larger: -384 or -128?~
