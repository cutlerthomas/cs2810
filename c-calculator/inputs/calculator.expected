6 + 7 = 13
-34 + 102314 = 102280
I do not understand the question, sorry!
7 - 5 = 2
-5 - -12 = 7
I do not understand the question, sorry!
12 * 12 = 144
3 * 97 = 291
-4 * -8 = 32
4 * -3 = -12
0 * 2 = 0
I do not understand the question, sorry!
100 / 10 = 10
256 / 8 = 32
256 / 9 = 28
3 / 4 = 0
4 / 4 = 1
Division by zero is undefined
0 / 15 = 0
I do not understand the question, sorry!
I do not understand the question, sorry!
20 % 7 = 6
7 % 20 = 7
-56 % -17 = -5
56 % -17 = 5
Division by zero is undefined
0 % 3 = 0
I do not understand the question, sorry!
-(16) = -16
-(-99) = 99
-(0) = 0
I do not understand the question, sorry!
(16)^2 = 256
(-99)^2 = 9801
(0)^2 = 0
I do not understand the question, sorry!
1000 is larger than 100
999 is larger than 9
Both values are the same
-128 is larger than -384
I do not understand the question, sorry!
